### DISCUSSION TITLE
The file doesn't exist

### DISCUSSION THREAD

**Discussion by f1scord** [Yue-Yang/ChatGPT-Siri#67] [General]
My API key is correct but for some reason I get this error with "file doesn't exist" ![image](https://user-images.githubusercontent.com/123882650/226214762-153d15d0-510b-4d50-9244-e8ef8b375d2c.png)![image](https://user-images.githubusercontent.com/123882650/226214764-08a2a446-e307-4e09-9ba7-743ec7cfc34d.png)![image](https://user-images.githubusercontent.com/123882650/226214769-c167f09d-4a3d-497e-935e-48f37a6953b8.png)

**Comment by Yue-Yang** [comment]
Can't find file, txt file not exits
You need to open 'Settings' - Apple ID - iCloud - iCloud Drive function to enable the feature of saving to txt file.

**Comment by savager** [comment]
There is no where else to save the text? I don't want to use icloud drive, or even enable it.
